### ISSUE TITLE
error certificate signed by unknown authority while docker pull ok

### ISSUE THREAD

**Issue by osallou** [appc/docker2aci#144]
Hi,
I try to convert an image from a private repository based on Docker registry v2.
A docker pull works fine, but when executing against docker2aci I have a *certificate signed by unknown authority* error.

    # bin/docker2aci docker://docker-registry.genouest.org/osallou/testgit
    Error: conversion error: Get https://docker-registry.genouest.org/v2: x509: certificate signed by unknown authority

Via Docker:

    # docker pull docker-registry.genouest.org/osallou/testgit
    latest: Pulling from docker-registry.genouest.org/osallou/testgit
    1986b9e4a6af: Already exists
    ....

**Comment by jsoriano** [comment]
@osallou your docker is probably configured with your CA, or to accept insecure registries.

Did you try to run docker2aci with the --insecure flag? I guess it should work, but with this I'm having an error getting repository data instead of the plain conversion error:
```
Error: conversion error: error getting repository data: Get https://.../v1/repositories/.../images: x509: certificate signed by unknown authority
```

(Yes, Docker registry v1 in my case)

**Comment by iaguis** [comment]
Fixes by #144, please reopen if it's still a problem. Thanks!
